custom_components/smartthinq_sensors/wideq/ac.py

```diff
@@ -648,6 +655,20 @@ async def get_power(self):
             self._current_power_supported = False
             return 0
 
+    async def set_mode_jet(self, status):
+        """Set the mode jet."""
+        keys = self._get_cmd_keys(CMD_STATE_MODE_JET)
+        if status:
+            if self._status.operation_mode is ACMode.HEAT.name:
+                jet = MODE_JET_HEAT
+            elif self._status.operation_mode in (ACMode.COOL.name, ACMode.DRY.name):
+                jet = MODE_JET_COOL
+            else:
+                return False
```

Comment:
Raise a error instead of just return:

```
raise ValueError("Invalid device status for jet mode")
```

Reply:
Done! I had thought about it, but I gave up so as not to generate errors in the log file due to a device condition that may be normal.